nautobot/extras/tests/test_management.py

```diff
@@ -55,3 +59,29 @@ def test_populate_role_choices_idempotent(self):
 
         populate_role_choices(apps=apps, schema_editor=None)
         self.assertEqual(Role.objects.count(), initial_roles_count)
+
+
+class JobManagementTestCase(TestCase):
+    """Test for the nautobot-server runjob helper function."""
+
+    def test_validate_job_and_job_data(self):
+        """
+        Test the validate_job_and_job_data function.
```

Comment:
You're not actually calling that function anywhere that I see though?

Reply:
The only thing we do to `job_data` in `validate_job_and_job_data()` is calling `json.loads()`. I am not sure comment on the next line is clear or not. Let me know if should be clearer.

Reply:
I feel like the test as written isn't doing anything particularly useful/valuable (we should be able to assume that json.dumps/json.load work as expected). Probably don't need to add/keep this test.